### PR TITLE
Save survey at server

### PR DESCRIPTION
**Pull request by SarahhhC**
Now we save survey at server...

@nlpenguin @ethkim @ace0625 @KyuaKwon

### CODE REVIEW

File: src/server/config/json/survey.data.js [hunk 1]
```diff
@@ -1,35 +0,0 @@
-/*
- * Sample Data for survey test
- */
-
-export default {
```
Comment (ethkim) [inline src/server/config/json/survey.data.js]: why is this file removed?

File: src/server/controllers/survey.controller.js [hunk 2]
```diff
@@ -31,9 +33,20 @@ export function getRequest(req, res, next) {
         }
       })
       .then((surveyItem) => {
-        if (surveyItem)
+        if (surveyItem) {
           res.status(200).json(surveyItem);
-        else {
+        } else {
+          if (surveyId == 'A001-1') {
```
Comment (ethkim) [inline src/server/controllers/survey.controller.js]: 1. Is this for mock survey? Or for the hard-coded survey?

2. Also, there should be more typed way to turn 'A001-1' into menteeData.surveyA001_1. Then you wouldn't need this if..else if... and just take care of it inline (turn string into enum type?)

3. Why is surveyId "A001-1" but enum name is "surveyA001_1"? (One is hypen "-", the other is underscore "_"). Please keep them consistent.

File: src/server/config/json/survey.data.js (hunk 1 above)
Comment (SarahhhC) [inline src/server/config/json/survey.data.js]: This file was just test data, so replaced this as real data.

File: src/server/controllers/survey.controller.js (hunk 2 above)
Comment (SarahhhC) [inline src/server/controllers/survey.controller.js]: This is the hard-coded survey...

**Comment by ethkim** [comment]
lgtm

**Comment by KyuaKwon** [comment]
:shipit:
